bundles/AdminBundle/src/Controller/Admin/External/AdminerController.php

```diff
@@ -0,0 +1,280 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Commercial License (PCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ *  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ *  @license    http://www.pimcore.org/license     GPLv3 and PCL
+ */
+
+namespace Pimcore\Bundle\AdminBundle\Controller\Admin\External {
+    use Pimcore\Bundle\AdminBundle\Controller\AdminController;
+    use Pimcore\Controller\KernelControllerEventInterface;
+    use Pimcore\Tool\Session;
+    use Symfony\Component\HttpFoundation\Request;
+    use Symfony\Component\HttpFoundation\Response;
+    use Symfony\Component\HttpKernel\Event\ControllerEvent;
+    use Symfony\Component\HttpKernel\Profiler\Profiler;
+    use Symfony\Component\Routing\Annotation\Route;
+
+    /**
+     * @internal
+     */
+    class AdminerController extends AdminController implements KernelControllerEventInterface
```

Comment:
please kick it. see https://github.com/pimcore/pimcore/pull/13921

Reply:
@dvesh3 done